tempora/schedule.py

```diff
@@ -23,21 +23,31 @@
 datetime.datetime(...utc)
 """
 
-import datetime
-import numbers
+from __future__ import annotations
+
 import abc
 import bisect
+import datetime
+import numbers
+from typing import TYPE_CHECKING
+
+from .utc import fromtimestamp as from_timestamp
+from .utc import now
 
-from .utc import now, fromtimestamp as from_timestamp
+if TYPE_CHECKING:
+    from typing_extensions import Self
 
 
 class DelayedCommand(datetime.datetime):
     """
     A command to be executed after some delay (seconds or timedelta).
     """
 
+    delay: datetime.timedelta = datetime.timedelta()
+    target: datetime.timedelta = datetime.timedelta()
```

Comment:
Oh, I missed this earlier - `target` is not a timedelta, but an `Any` (probably a `Callable[..., Any]`, if used with the `InvokeScheduler`, but could be anything if used with the `CallbackScheduler`).